cs-offerings/scripts/delete/delete_chaincode-install.sh

```diff
@@ -21,12 +21,12 @@ echo "Preparing yaml for chaincodeinstall pod"
 sed -e "s/%PEER_ADDRESS%/${PEER_ADDRESS}/g" -e "s/%PEER_MSPID%/${PEER_MSPID}/g" -e "s|%MSP_CONFIGPATH%|${MSP_CONFIGPATH}|g"  -e "s/%CHAINCODE_NAME%/${CHAINCODE_NAME}/g" -e "s/%CHAINCODE_VERSION%/${CHAINCODE_VERSION}/g" ${KUBECONFIG_FOLDER}/chaincode_install.yaml.base > ${KUBECONFIG_FOLDER}/chaincode_install.yaml
 
 echo "Deleting Existing Install Chaincode Pod"
-if [ "$(kubectl get pods | grep chaincodeinstall | wc -l | awk '{print $1}')" != "0" ]; then
+if [ "$(kubectl get pods chaincodeinstall | grep chaincodeinstall | wc -l | awk '{print $1}')" != "0" ]; then
```

Comment:
Why is this change required?

Reply:
I kept it consistent with the version that was committed in the master branch by rddill. I also observed that $(kubectl get pods | grep chaincodeinstall  wasn't working. I had to use $(kubectl get pods -a | grep chaincodeinstall or $(kubectl get pods chaincodeinstall for this to work

Reply:
thanks